### PR TITLE
reccard: convert Gtk2 calls to Gtk3

### PR DESCRIPTION
**Pull request by cydanil**
Along with the other PRs, this finally allows the display of the record card.

The changes here are mostly updates in API calls for Gtk3.  

At this stage, it's the tests (#32 ) for it that crash, and not the application itself anymore.

It's most likely not the last PR for it, but now I can focus on updating the tests, and then we'll be able to check its behaviour! :)

### CODE REVIEW

File: gourmet/reccard.py [hunk 1]
```diff
@@ -1422,15 +1422,15 @@ def update_from_database (self):
             debug("Widget for %s"%c,5)
             model = self.rg.get_attribute_model(c)
             self.rw[c].set_model(model)
-            self.rw[c].set_text_column(0)
-            cb.setup_completion(self.rw[c])
+            # self.rw[c].set_text_column(0)
+            # cb.setup_completion(self.rw[c])
```
Comment (maweki) [inline gourmet/reccard.py]: Have we (temporarily) removed completion here? Can you add a comment why and what's the problem.

Comment (cydanil) [reply gourmet/reccard.py]: That should not be there, good catch

Comment (cydanil) [reply gourmet/reccard.py]: Fixed: the change is two-fold: first the API call has changed for setting the text column, but also the API call for getting the children when setting the callbacks.

**Review by maweki — APPROVED** [review]

**Comment by cydanil** [comment]
Thanks for the reviews!

**Pull request merged**